docs/getting-started/what-is-dotnet.md

```diff
@@ -30,7 +30,7 @@ To get a better sense of what .NET Core is, let’s compare it to the.NET Framew
 </td>
 </tr>
 <tr>
-<td>Can be used to develop apps that take advantage of a number of technologies, such as ASP .NET Core, for building web applications; Windows Communication Foundation (WCF), for building services, including SOAP; and Workflow Foundation (WF), for building workflows. 
+<td>Can be used to develop apps that take advantage of a number of technologies, such as ASP .NET Core, for building web applications; Windows Communication Foundation (WCF), for building applications that communicate with existing WCF services; and Workflow Foundation (WF), for building workflows. 
```

Comment:
I'm not clear on what distinguishes the two sides of this row in the table. What's really different about .NET Core here? If there's no real difference, is this row necessary?